src/Http/Controllers/UploadController.php

```diff
@@ -42,7 +43,16 @@ protected function init($request)
     {
         $resource = $request->findResourceOrFail();
 
-        $this->tool = collect($resource->availablePanelsForDetail($request, $resource))
+        $this->model = $resource->model();
```

Comment:
Might be a copy paste error here; UploadController has no model property (still works though, but unnecessary)